test/lib/ceych.js

```diff
@@ -154,4 +155,99 @@ describe('ceych', () => {
       sinon.assert.called(cacheClient.stop);
     });
   });
+
+  describe('.invalidate', () => {
+    it('invalidates the cache entry', (done) => {
+      const func = ceych.wrap(wrappable);
+
+      func((err, result) => {
+        assert.ifError(err);
+        assert.equal(result, 1);
+        sinon.assert.calledOnce(wrappable);
+
+        // Invalidate the cache entry
```

Comment:
You can probably drop the comments with the exception of the more complex test (like the last one).